returnn/torch/engine.py

```diff
@@ -366,6 +366,10 @@ def train_epoch(self):
         extern_data = None
         num_seqs = None
         last_seq_idx = 0
+
+        ep_data_len = NumbersDict()
+        ep_data_space = NumbersDict()
```

Comment:
The var names are not good. I don't understand the meaning of "len" or "space". Also, the `ep_` prefix is inconsistent to the other var names here (e.g. `num_seqs`).
```suggestion
        total_data_size_packed = NumbersDict()
        total_data_size_padded = NumbersDict()
```